src/test/groovy/io/seqera/wave/controller/RegistryControllerRedisTest.groovy

```diff
@@ -114,7 +114,7 @@ class RegistryControllerRedisTest extends Specification implements DockerRegistr
         HttpClient client = applicationContext.createBean(HttpClient)
         and:
         jedis.set("wave-tokens/v1:1234", '{"containerImage":"library/hello-world"}')
-        jedis.set("wave-build/v1:library/hello-world", '{"containerImage":"library/hello-world"}')
+        jedis.set("wave-build/v2:library/hello-world", '{"request":{"buildId":"1234","containerImage":"library/hello-world","startTime":"2021-09-29T15:00:00Z"},"result":{"containerImage":"library/hello-world","id":"1234","exitStatus":1}}')
```

Comment:
This is too tricky to maintain. It would be better to use `BuildStore` API to populate it

Reply:
ok sure, i will change it

Reply:
I'm going to merge this PR. Let's address this in the upstream branch

Reply:
ok